Project_Xml/ArgumentXml.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Project_Interface;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Project_Xml
+{
+    public class ArgumentXml : IFileHandling
+    {
+        string[] _clArguments = null;
+
+        public ArgumentXml(string[] clArguments)
```

Comment:
`Scope`: This is a class that parses data.  Should it really need to know about command line arguments?